setup.py

```diff
@@ -8,6 +8,7 @@
     raise RuntimeError("aiohttp 4.x requires Python 3.9+")
 
 
+USE_SYSTEM_DEPS: bool = bool(os.environ.get("USE_SYSTEM_DEPS"))
```

Comment:
Not sure why the other one has an annotation, but it's rather explicitly a bool, let's drop it.
```suggestion
USE_SYSTEM_DEPS = bool(os.environ.get("USE_SYSTEM_DEPS"))
```